installation/docker/dockercompose.yml

```diff
@@ -32,6 +37,13 @@ services:
       replicas: 1
       restart_policy:
         condition: on-failure
+    volumes:
+      - type: bind
+        source: /var/run/docker.sock
+        target: /var/run/docker.sock
+      - type: bind
+        source: /tmp
+        target: /tmp
```

Comment:
There isn't a particularly clean way to do this in compose, but it would be nice if this was optional based on the chosen provider.